src/views/home/components/Borrow.tsx

```diff
@@ -0,0 +1,78 @@
+import { FC, useState } from "react";
+import { useWallet } from "@solana/wallet-adapter-react";
+import AssetInput from "./AssetInput";
+import { BNumber, BZero, max, min } from "utils/utils";
+import ConfirmButton from "./ConfirmButton";
+import MoreParams from "./MoreParams";
+type BorrowPropsType = {
+  selectedToken: any;
+  onFinish: () => void;
+};
+
+const Borrow: FC = ({ selectedToken, onFinish }: BorrowPropsType) => {
+  const { publicKey } = useWallet();
+  const [amount, setAmount] = useState();
+  const [value, setValue] = useState<string | null>(null);
+  const [showFees, setShowFees] = useState<boolean>(true);
+  const [showRewards, setShowRewards] = useState<boolean>(false);
+  const [showPositions, setShowPositions] = useState<boolean>(false);
+
+  const valueObj = value ? new BNumber(value) : null;
+
+  const onValueChange = (val: string | null) => {
+    setShowPositions(true);
+    setValue(val);
+  };
+  //TODO:get max value, remove dummy data
+  const maxValue = BZero;
+
+  const [disableConfirmButton, buttonText] = ((): [boolean, string] => {
+    if (!publicKey) {
+      return [false, "Connect wallet"];
+    }
+    return [false, "Enter a value"];
+  })();
+
+  const onClick = async (val: string) => {};
+  return (
+    <div className="flex flex-col ">
+      <AssetInput
+        selectedToken={selectedToken}
+        onChange={onValueChange}
+        value={value}
+        maxPossibleValue={max(BZero, maxValue).toString()}
+      />
+      <div className="flex flex-col gap-2 py-4">
+        <span className="flex justify-between">
+          <h3 className="text-neutral-content text-sm">User borrow limit</h3>
```

Comment:
It would be nice to have a lot of common styles that correspond to a design style show up in globals.css or even typography.css. Then here, we would just say `className="solend-label"` or something or `className="solend-primary"`

Basically all the elements of the [solend design system](https://www.figma.com/file/tjyfF5smzMSfXIxjvIfU7D/Solend-%E2%80%93-App-%26-Site?node-id=3001%3A130578) can be global styles and not copied throughout the app

Reply:
maybe there's a cool tailwind way to do this too

Reply:
If you look in the web repo I sent you, The Typography component is a great example of DRYing up a lot of the font styles